runtime/ftplugin/help.lua

```diff
@@ -2,6 +2,27 @@
 vim.treesitter.start()
 
 -- add custom highlights for list in `:h highlight-groups`
-if vim.endswith(vim.fs.normalize(vim.api.nvim_buf_get_name(0)), '/doc/syntax.txt') then
-  require('vim.vimhelp').highlight_groups()
+local bufname = vim.fs.normalize(vim.api.nvim_buf_get_name(0))
+if vim.endswith(bufname, '/doc/syntax.txt') then
+  require('vim.vimhelp').highlight_groups({
+    { start = [[\*group-name\*]], stop = '^======', match = '^(%w+)\t' },
```

Comment:
I don't want to copy-paste the same `stop` pattern every time there's a change, so consider creating a variable.

Reply:
If there's a change, it won't necessary be the same for all `stop` patterns.

Reply:
Fair point, but given that these represent section delimiters I do forecast some consistency (even in the event of change). I'm just not a fan of random string literals, but I won't die on a hill for this.

Reply:
They _are_ random, though; the just happen to be the same in half the cases here (and could even be tightened up in some of them, which I've thought about doing).